src/main/java/Storage.java

```diff
@@ -0,0 +1,170 @@
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.nio.file.*;
```

Comment:
The ordering of your import statements can be more consistent (eg. label all java.io first, followed by java.nio and then java.util).